Yosemite/YosemiteTests/Stores/ShippingLabelStoreTests.swift

```diff
@@ -38,6 +38,130 @@ final class ShippingLabelStoreTests: XCTestCase {
         super.tearDown()
     }
 
+    // MARK: - `loadShippingLabels`
+
+    func test_loadShippingLabels_persists_shipping_labels_and_settings_on_success() throws {
+        // Given
+        let remote = MockShippingLabelRemote()
+        let orderID: Int64 = 22
+        let expectedOriginAddress = Yosemite.ShippingLabelAddress(company: "fun testing",
```

Comment:
I removed all the `Yosemite.*` declarations and the test still builds. I think they're not needed. 🤷 

```suggestion
        let expectedOriginAddress = ShippingLabelAddress(company: "fun testing",
```

Reply:
Removed with your suggested changes here: https://github.com/woocommerce/woocommerce-ios/pull/3176/commits/344ec72aaff55eb55026eacd893be18dbc55aa2d